apps/prairielearn/src/migrations/20230807222115_stripe_checkout_sessions__create.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS
+  stripe_checkout_sessions (
+    id BIGSERIAL PRIMARY KEY,
+    -- TODO: better name?
+    session_id TEXT NOT NULL UNIQUE,
+    institution_id BIGINT REFERENCES institutions (id) ON UPDATE CASCADE ON DELETE CASCADE,
+    course_instance_id BIGINT REFERENCES course_instances (id) ON UPDATE CASCADE ON DELETE CASCADE,
+    user_id BIGINT NOT NULL REFERENCES users (user_id) ON UPDATE CASCADE ON DELETE CASCADE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    completed_at TIMESTAMPTZ,
+    data JSONB NOT NULL,
+    plan_names TEXT[] NOT NULL,
+    plan_grants_created BOOLEAN NOT NULL DEFAULT FALSE
+  );
+
+CREATE INDEX IF NOT EXISTS stripe_checkout_sessions_institution_course_instance_user_idx ON stripe_checkout_sessions (institution_id, course_instance_id, user_id);
```

Comment:
It might be? I was following the same pattern as tables like `plan_grants` where we index on everything in our object hierarchy. Maybe we shouldn't actually be doing that? Or maybe we simply don't need to in this situation? Maybe we should chat about this on zoom 😅

Reply:
I dropped this index in aca0da7.